barretenberg/cpp/src/barretenberg/vm2/constraining/relations/instr_fetching.test.cpp

```diff
@@ -614,9 +603,8 @@ TEST(InstrFetchingConstrainingTest, NegativeWrongWireInstructionSpecInteractions
             // We do not need to re-run LookupIntoIndexedByClk<wire_instr_spec_lookup::Settings>().process(trace);
```

Comment:
@jeanmon I disabled 2 tests because of a comment like this. I'll leave it to you to think if the test still makes sense.